org.sailfishos.usermanager.conf

```diff
@@ -22,5 +22,6 @@ Properties"/>
     <allow send_destination="org.sailfishos.usermanager" send_interface="org.sailfishos.usermanager" send_member="removeUser" />
     <allow send_destination="org.sailfishos.usermanager" send_interface="org.sailfishos.usermanager" send_member="modifyUser" />
     <allow send_destination="org.sailfishos.usermanager" send_interface="org.sailfishos.usermanager" send_member="users" />
+    <allow send_destination="org.sailfishos.usermanager" send_interface="org.sailfishos.usermanager" send_member="switchUser" />
```

Comment:
This is still switchUser and currentUser is missing.

Reply:
Doh, too many places, fixed.